services/web/server/src/simcore_service_webserver/resource_manager/garbage_collector.py

```diff
@@ -69,6 +72,29 @@ async def collect_garbage(registry: RedisResourceRegistry, app: web.Application)
                     app=app,
                 )
 
+                await remove_resources_if_guest_user(
+                    app=app, project_uuid=resource_value, user_id=int(key["user_id"])
+                )
+
+
+async def remove_resources_if_guest_user(
+    app: web.Application, project_uuid: str, user_id: int
+) -> None:
+    """When a guest user finishes using the platform its Posgtres
+    and S3/MinIO entries need to be removed
```

Comment:
where do you remove s3/minio entries??

Reply:
the `delete_project_from_db` has a call to [delete_data_folders_of_project](https://github.com/ITISFoundation/osparc-simcore/blob/4c2ed18390a730c3c67c7f477c66681d9da4be54/services/web/server/src/simcore_service_webserver/projects/projects_api.py#L182)